crowbar_framework/config/environments/production.rb

```diff
@@ -41,10 +41,10 @@
 
   config.assets.debug = false
   config.assets.raise_runtime_errors = true
-  config.assets.js_compressor = :uglifier
+  config.assets.js_compressor = :closure
   config.assets.css_compressor = :sass
-  config.assets.compile = true
-  config.assets.digest = true
```

Comment:
Hrm. This is wrong, as this makes the pages reference the compiled assets directly -- that won't solve the issue with the static pages (which cannot reference the compiled assets).

So I'm stuck with a routing errors:
`ActionController::RoutingError (No route matches [GET] "/assets/barclamps/crowbar/application.js")`